extra-python/hstspreload/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=hstspreload
+PKGSEC=python
+PKDDEP="python-3"
+BUILDDEP="setuptools"
+PKGDES="Chromium HSTS Preload list"
```

Comment:
This description is not exactly descriptive. Although summarised as such on its original [PyPI page](https://pypi.org/project/hstspreload/) (which I suspect is where you got this from). I would suggest changing this.

Reply:
789c19a